test_scripts/API/VehicleData/Refactor_Fuel_Information_Related_Vehicle_Data/001_Get_Subscribe_UnsubscribeVehoricleData_by_fuelRange.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal:https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0256-Refactor-Fuel-Information-Related-Vehicle-Data.md
+-- Description: SDL successfully process RPCs GetVehicleData, SubscribeVehicleData, UnsubscribeVehicleData
+-- with all parameters of structure `FuelRange
+-- In case:
+-- 1) App sends Get/Subscribe/UnsubscribeVehicleData(fuelRange:true) request
+-- 2) SDL transfers this request to HMI
+-- 3) HMI sends all parameters of structure `FuelRange`
+--    (type, range, level, levelState, capacity, capacityUnit)
+-- SDL does:
+-- 1) send Get/Subscribe/UnsubscribeVehicleData response to mobile with all parameters in `FuelRange` structure
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
```

Comment:
Please remove runner from script and use common file instead.
Comment is related to all scripts.